app/ui/browser-modern/sagas/ui-sagas.js

```diff
@@ -70,12 +70,12 @@ function* setURLBarValue({ urlbar, value, doc, options = { keepSelection: true }
   }
 }
 
-function* showDownloadNotification({ filename, status }) {
+function* showDownloadNotification({ path, filename, status }) {
   let message;
   const title = `Download ${status === 'success' ? 'completed' : 'failed'}`;
 
   if (status === 'success') {
-    message = `Download for ${filename} is completed.`;
+    message = `Download for ${filename} is completed. Can be located at ${path}/${filename}`;
```

Comment:
Additionally, it might be useful to make this path relative to the user's home directory to save up even more space. Like, show `~/Downloads` instead of the full path to the user's downloads folder.